eskip/eskip_test.go

```diff
@@ -239,19 +239,39 @@ func TestParseFilters(t *testing.T) {
 		"error",
 		"trallala",
 		nil,
-		// TODO: fix position
-		"parse failed after token ->, position 16: syntax error",
+		"parse failed after token trallala, position 8: syntax error",
 	}, {
 		"error 2",
 		"foo-bar",
 		nil,
-		// TODO: fix position
-		"parse failed after token foo, position 8: syntax error",
+		"parse failed after token foo, position 3: syntax error",
 	}, {
 		"success",
 		`filter1(3.14) -> filter2("key", 42)`,
 		[]*Filter{{Name: "filter1", Args: []interface{}{3.14}}, {Name: "filter2", Args: []interface{}{"key", float64(42)}}},
 		"",
+	}, {
+		"a comment produces nil filters without error",
+		"// a comment",
```

Comment:
We can disallow comments in ParsePredicates and ParseFilters, needs a decision.

Reply:
Depends on if we would have a benefit from that. I guess it won't be much and then I would just let it like that.

Reply:
Since we added https://github.com/zalando/skipper/pull/2905 I think it makes sense to allow eskip comments for ParsePredicates and ParseFilters